src/main/java/net/coderbot/iris/Iris.java

```diff
@@ -154,7 +155,11 @@ private static boolean loadExternalShaderpack(String name) {
 				.filter(path -> path.endsWith("shaders"))
 				.findFirst();
 		} catch (IOException e) {
-			logger.error("Error while finding shaderpack for zip directory {}", shaderpackPath);
+			if (e instanceof ZipException && e.getMessage().equals("zip END header not found")) {
+				logger.error("The shaderpack appears to be corrupted, please try downloading it again {}", shaderpackPath);
```

Comment:
The message is good but I wonder if it will be too hard for some people to spot. Maybe the shader pack GUI could have a better error dialog to go along with this?